appveyor.yml

```diff
@@ -22,30 +22,33 @@ only_commits:
 skip_tags: true
 
 # At most one job during this build
-max_jobs: 1
+#max_jobs: 1
```

Comment:
You can remove things instead of commenting out, we're on version control and can see previous versions of files.